src/library.js

```diff
@@ -642,11 +642,30 @@ LibraryManager.library = {
 
   system__deps: ['__setErrNo'],
   system: function(command) {
+#if ENVIRONMENT_MAY_BE_NODE
+    if (ENVIRONMENT_IS_NODE) {
+      if (!command)
+        return 1; // shell is available
+
+      let cp = require('child_process');
```

Comment:
My understanding is that we currently avoid ES6 features in the emscripten code.   So for example we don't yet allow `let` in the codebase.

Reply:
In general, yes, because IE11 doesn't have `let` support. However, in this case we just care about node so it seems fine to use `let` I think. But, probably we should have some clear policy on this, so for now I agree, let's use `var` here.

Reply:
I'll change it to var

Reply:
Does that mean `const` is fine? IE11 supports that.

Reply:
@VirtualTim In principle `const` could be fine - it is larger, though, which is unfortunate. I'm not sure if minifiers optimize them into `vars`? If they do, maybe we can use it for clarity.

Reply:
Ill keep that in mind next time.
I wonder if there is any performance improvements in using const over var? The JS compiler may be able to do something smarter if it know a variable is read only?
Also at the end of the day I think we can assume that everyone is gziping their JS, so I don't imagine that adding const will add any meaningful size increase anyway.

Reply:
You'd be surprised how much ppl case about size.  We do some pretty horrible things in our JS code to save a couple of bytes.   Using `var` over `const` is one of the least egregious :).

Also, I believe the using `const` is JS, while great for readability and maintainabltiy of source, has basically no run time impact.   In fact we are kind of hoping that closure might re-write all the const values to save a few bytes if we ever did allow it at the source level.

Reply:
Yeah, I think once you get to the VM it will do SSA analysis anyhow of the JS variables, so it checks if they are constant or not, regardless of how they are declared. Maybe it matters somehow in a baseline compiler, though, so maybe it affects startup, but I doubt it's significant.